ci-operator/step-registry/cucushift/installer/check/azure/private/lb/cucushift-installer-check-azure-private-lb-commands.sh

```diff
@@ -0,0 +1,128 @@
+    az cloud update --profile 2019-03-01-hybrid
+else
+    az cloud set --name AzureCloud
+fi
+az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none
+
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+RESOURCE_GROUP=$(yq-go r "${INSTALL_CONFIG}" 'platform.azure.resourceGroupName')
+if [[ -z "${RESOURCE_GROUP}" ]]; then
+    RESOURCE_GROUP="${INFRA_ID}-rg"
+fi
+
+publish=$(yq-go r "${INSTALL_CONFIG}" "publish")
+outbound_type=$(yq-go r "${INSTALL_CONFIG}" "platform.azure.outboundType")
+echo -e "--- cluster config --- \npublish: ${publish}\noutbound_type: ${outbound_type}"
+
+if [[ "${publish}" != "Internal" ]]; then
+    echo "This step is applicable on azure private cluster!"
+    exit 1
+fi
+
+critical_check_result=0
+no_critical_check_result=0
+
+#lb check
+lb_list=$(az network lb list -g ${RESOURCE_GROUP} -ojson |jq -r '.[].name')
+echo "INFO: lb list is ${lb_list}"
+internal_lb="${INFRA_ID}-internal"
+if [[ "${lb_list}" =~ ${internal_lb} ]]; then
+    public_lb=$(echo ${lb_list/"${internal_lb}"} | tr -d '\n')
+else
+    echo "ERROR: unable to find internal lb ${internal_lb}!"
+    exit 1
+fi
+
+if [[ "${outbound_type}" == "UserDefinedRouting" ]]; then
+    echo "Cluster outBound type is UserDefinedRouting, this is fully private cluster..."
+    if (( ocp_minor_version >= 11 )); then
+        echo "Check that no public load balancer created on fully private cluster..."
+        [[ -n "${public_lb}" ]] && echo "ERROR: Found public load balancer ${public_lb} on fully private cluster!" && no_critical_check_result=1
+    fi
+    # check lb frontended IP config
+    for lb in ${lb_list}; do
+        echo "Check public ip address of Loadbalance ${lb}, should not be created..."
+        lb_public_ip=""
+        lb_public_ip=$(az network lb show -g ${RESOURCE_GROUP} -n ${lb} | jq -r '.frontendIPConfigurations[].publicIPAddress.id' | grep -v 'null')
+        [[ -n ${lb_public_ip} ]] && echo "ERROR: Found public ip address ${lb_public_ip} for load balancer: ${lb}" && critical_check_result=1
+    done
+else
+    echo "Cluster outBound type is loadBalancer, this is private cluster..."
+    echo "Check that outbound rules are created on private cluster..."
+    outbound_rules=$(az network lb show --name ${public_lb} -g ${RESOURCE_GROUP} -ojson | jq -r ".outboundRules[]")
+    if [[ -z "${outbound_rules}" ]]; then
+        echo "ERROR: Not found outbound rules for public load balancer ${public_lb} on private cluster!"
+        critical_check_result=1
+    else
+        echo -e "Found outbound rules for public lb ${public_lb}\n${outbound_rules}"
+    fi
+
+    echo "Check public ip address of Loadbalance ${public_lb}, should be configured..."
+    lb_public_ip=""
+    lb_public_ip=$(az network lb show -g ${RESOURCE_GROUP} -n ${public_lb} | jq -r '.frontendIPConfigurations[].publicIPAddress.id') || true
```

Comment:
`|| true` is placed to a wrong line? should be line 97 ?

Reply:
you are right, fixed.